smdebug/tensorflow/version_check.py

```diff
@@ -1,11 +1,11 @@
 # Third Party
-import tensorflow as tf
+import tensorflow.compat.v1 as tf
 from packaging import version
 
 # Separating this out to a different file because black doesn't like any code before imports
 # So this can't go at the top of init.py
 # If this is not at the top, then we see unhelpful messages as we fail to import some modules
-if version.parse(tf.__version__) >= version.parse("2.0.0") or version.parse(
-    tf.__version__
-) < version.parse("1.13.0"):
-    raise ImportError("SMDebug only supports TensorFlow 1.13.x, 1.14.x and 1.15.x")
+if version.parse(tf.__version__) < version.parse("1.14.0"):
+    raise ImportError(
+        "SMDebug only supports TensorFlow 1.14, 1.15, and experimental functionality in 2.x."
```

Comment:
why?

Reply:
Per the comments above, we are introducing support for TF 2.x where 1.13 was not supported with ZCC. Based on a customer feedback we can introduce support for 1.13 later, however at this point it is clear that more and more customers are migrating towards TF 2.x so it doesn't make sense to prioritize fixing of 1.13.

Reply:
raising will cause training job to halt. Will customers using TF 1.13.0 in Sagemaker will start seeing job failures all of a sudden, if there is possibility then it will be seen as regression.